gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Sat Apr 25 15:02:11 PDT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
```

Comment:
This isn't truly in the scope of this branch, but IntelliJ said it would work better on gradle inspections if it had the sources, so I let it make that change.